public/app/features/connections/tabs/ConnectData/CardGrid/CardGrid.tsx

```diff
@@ -38,6 +32,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
         "Figure Description Actions"
         "Figure    Meta     Actions"
         "Figure     -       Actions"`,
+    marginBottom: '0px !important',
```

Comment:
Why is this needed?

Reply:
`Card` applies a margin-bottom that makes the spacing inconsistent so I got rid of it while working on the `Grid` but forgot to delete this line. Thanks for telling me ;)